public/js/pimcore/elementservice.js

```diff
@@ -83,9 +83,19 @@ pimcore.elementservice.deleteElementsComplete = function(options, response) {
 pimcore.elementservice.deleteElementCheckDependencyComplete = function (window, res, options) {
 
     try {
-        let message = res.batchDelete ? t('delete_message_batch') : t('delete_message');
+        let message = '';
+        if (res.batchDelete) {
+            message += sprintf(t('delete_message_batch'), res.itemResults.length) + "<br /><div style='display: block; padding-top: 10px'>";
```

Comment:
```suggestion
            message += sprintf(t('delete_message_batch'), res.itemResults.length) + "<br /><div>";
```
`div` tags are already `block`, i'd prefer to let `ul` tag set the paddings (see the other feedback below)